### DISCUSSION TITLE
[Question]: My Question is how do i fix this?

### DISCUSSION THREAD

**Discussion by Prrimcord** [danny-avila/LibreChat#2166] [Troubleshooting]
### What is your question?

![Screenshot 2024-03-23 004356](https://github.com/danny-avila/LibreChat/assets/103951992/41c2146c-9091-49e2-83a2-eaa9db8cb09d)

### More Details

whenever i try to use it, it gives me this error and how do i get the assistant api key?
![Screenshot 2024-03-23 004801](https://github.com/danny-avila/LibreChat/assets/103951992/73842db5-ff82-4e6a-8e4b-5180da626037)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by fuegovic** [comment]
looks like you might need to add more credit into your account:
https://platform.openai.com/account/billing/overview
